examples/pytorch/README.md

```diff
@@ -23,6 +23,9 @@ python3 quickstart_advanced.py --model_dir nvidia/Llama-3.1-8B-Instruct-FP8 --tp
 
 # FP8(e4m3) kvcache
 python3 quickstart_advanced.py --model_dir nvidia/Llama-3.1-8B-Instruct-FP8 --kv_cache_dtype fp8
+
+# Nemotron-H requires block reuse to be disabled in kvcache
+python3 quickstart_advanced.py --model_dir nvidia/Nemotron-H-8B-Base-8K --kv_cache_enable_block_reuse --max_batch_size 8
```

Comment:
I find that it's a bit of misleading that `--kv_cache_enable_block_reuse` disable KV cache reuse. Could you please make a change:

```python

    parser.add_argument('--disable_kv_cache_reuse',
                        default=False,
                        action='store_true')

    kv_cache_config = KvCacheConfig(
        enable_block_reuse=not args.disable_kv_cache_reuse,
        free_gpu_memory_fraction=args.kv_cache_fraction,
    )
```

Reply:
Done